plenty_api/api.py

```diff
@@ -837,6 +839,48 @@ def plenty_api_get_contacts(self,
             refine=refine,
             additional=additional)
 
+    def plenty_api_get_property_selections(self,
+                                           refine: dict = None,
+                                           additional: list = None):
+        """
+        Get maaping of selection IDs to the actual value in all avaiable
+        languages.
+
+        Parameter:
+            refine      [dict]  -   Apply filters to the request
+                                    Example:
+                                    {'email': 'a@posteo.net', 'name': 'Thomas'}
+            additional  [list]  -   Add additional elements to the response
+                                    data.
+                                    Example:
+                                    ['addresses']
+
+        Return:
+                        [dict]  -   Mapping of all selection property values
+        """
+        if self.data_format == 'dataframe':
+            logging.error("Mapping can only be created with the JSON format")
+            return
+
+        data = self.__plenty_api_generic_get(
+            domain='property',
+            path='/selections',
+            refine=refine
```

Comment:
The `/rest/properties/selections` route does not accept parameters, the way to refine the result is by using the `/rest/properties/selections` when no refine argument is defined and `​/rest​/v2​/properties​/{propertyId}​/selections` when one property ID is given. You can also think about accepting a list of property IDs and then say len(list) < 4 => loop through the single requests else request all and filter the IDs out.

Reply:
It actually does, at least in this implementation if you give the refine argument into the function {'propertyId': 121} it only requests and also only give you the values of the propertyId 121, must not be properly documented in the PlentyMarkets documentation, or it is what the call PropertySelectionId, but it still works.

Reply:
Alright, we should highlight that we diverge from the official documentation in this case within our documentation.